client/layout/universal-navbar-footer/index.tsx

```diff
@@ -155,68 +165,64 @@ const UniversalNavbarFooter = () => {
 									className="lp-language-picker__content"
 									title={ translate( 'Change Language' ) }
 									onChange={ ( e ) => navigate( e.target.value ) }
-									defaultValue={
-										translate.localeSlug === 'en'
-											? `/plugins`
-											: `/${ translate.localeSlug }/plugins`
-									}
+									defaultValue={ `${ window.location.pathname }` }
```

Comment:
We could simplify the string literal syntax here:

```suggestion
									defaultValue={ window.location.pathname }
```